sync2jira/intermediary.py

```diff
@@ -19,6 +19,13 @@
 from datetime import datetime
 import re
 
+import pypandoc
```

Comment:
Make sure to add this to the [DockerFile](https://github.com/release-engineering/Sync2Jira/blob/master/Dockerfile) if it needs to be pip installed.

Reply:
It's already installed using `dnf`. The package is used to convert github comments to plain text (based on the `github_markdown` option).